lib/forms/pegasus_form_validation.rb

```diff
@@ -4,6 +4,7 @@
 require_relative 'pegasus_form_errors'
 require_relative '../cdo/regexp'
 require_relative '../cdo/geocoder'
+require_relative '../../pegasus/forms/hoc_signup_2017'
```

Comment:
OIC. I think the form(s) should be required outside of `PegasusFormValidation`, in this case in your test. The reason it's complaining about the uninitialized constant `HocSignup2017` is because the test is [creating a form of that type](https://github.com/code-dot-org/code-dot-org/pull/16938/files#diff-046bf0d1021ea30f562b024992768939R69). 

Either the `test_form_helpers` test itself should require that form, or we should modify `PegasusFormHelpers` to take a class instead of kind (string), or otherwise provide a flexible way to supply the set of available forms, but I don't think it should require them directly. Part of my goal in extracting that code was to isolate it so it's more portable and testable.